applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_internal.h

```diff
@@ -8,40 +8,15 @@
 #define _BROADCAST_INTERNAL_H_
 
 #include "le_audio.h"
+#include "nrf5340_audio_common.h"
 
-#if CONFIG_BT_AUDIO_BROADCAST_CONFIGURABLE
-#define BT_BAP_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                  \
-	BT_BAP_LC3_PRESET_CONFIGURABLE(                                                            \
-		BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT,                      \
-		BT_AUDIO_CONTEXT_TYPE_MEDIA, CONFIG_BT_AUDIO_BITRATE_BROADCAST_SRC)
+#include <zephyr/zbus/zbus.h>
 
-#elif CONFIG_BT_BAP_BROADCAST_16_2_1
-#define BT_BAP_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                  \
-	BT_BAP_LC3_BROADCAST_PRESET_16_2_1(BT_AUDIO_LOCATION_FRONT_LEFT |                          \
-						   BT_AUDIO_LOCATION_FRONT_RIGHT,                  \
-					   BT_AUDIO_CONTEXT_TYPE_MEDIA)
+ZBUS_CHAN_DEFINE(le_audio_chan, struct le_audio_msg, NULL, NULL, ZBUS_OBSERVERS_EMPTY,
```

Comment:
Another solution would be to have all zbus related things like this in separate file(s)?
I think we have discussed this before